cli/command/swarm/ca.go

```diff
@@ -61,6 +61,10 @@ func runRotateCA(dockerCli command.Cli, flags *pflag.FlagSet, opts caOptions) er
 	}
 
 	if !opts.rotate {
+		if flags.Changed(flagCACert) || flags.Changed(flagCAKey) || flags.Changed(flagCertExpiry) {
+			fmt.Fprintln(dockerCli.Out(), "Warning: because the `--rotate` flag was not provided, all other flags will be ignored")
+		}
```

Comment:
As a user, if I run a command like 

```
docker swarm ca --ca-cert foo
```

I think I'm expecting an update, but the command acts as basically a "print the current value". That feels like an error to me.